Documentation/ceph-nfs-crd.md

```diff
@@ -66,6 +70,12 @@ spec:
     priorityClassName:
 ```
 
+ You can enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool with the following command in the ceph-tools container:
```

Comment:
maybe this looks better
```suggestion
 To enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool by running `$ ceph dashboard set-ganesha-clusters-rados-pool-namespace <pool_name>[/<namespace>]` in the toolbox container.
```

Reply:
Good suggestion, but remove `To` from the beginning.